pages/prediction-sets/[id].js

```diff
@@ -120,6 +120,62 @@ export default function PredictionSet({
                     <DataItemValue>{predictionSet.scope}</DataItemValue>
                   </>
                 )}
+                {predictionSet.small_scale_gene_list && (
+                  <>
+                    <DataItemLabel>Small Scale Gene List</DataItemLabel>
+                    <DataItemValue>
+                      <SeparatedList isCollapsible>
+                        {predictionSet.small_scale_gene_list.map((gene) => (
+                          <Link href={gene["@id"]} key={gene["@id"]}>
+                            {gene.symbol}
+                          </Link>
+                        ))}
+                      </SeparatedList>
+                    </DataItemValue>
+                  </>
+                )}
+                {predictionSet.small_scale_loci_list && (
+                  <>
+                    <DataItemLabel>Small Scale Loci List</DataItemLabel>
+                    <DataItemValue>
+                      <SeparatedList isCollapsible>
+                        {predictionSet.small_scale_loci_list.map((loci) => (
+                          <>
+                            {loci.assembly} {loci.chromosome} {loci.start}
+                            {"-"}
+                            {loci.end}
+                          </>
+                        ))}
+                      </SeparatedList>
+                    </DataItemValue>
+                  </>
+                )}
```

Comment:
We have a standard [chromosome location display component](https://github.com/IGVF-DACC/igvf-ui/blob/454abc9797003d968d3e3bdf225dfe423b0f468d/components/chromosome-locations.js#L39) you can use, both for single chromosome as well as lists.

Reply:
done